### ISSUE TITLE
How a cool idea!Why not continue it?

### ISSUE THREAD

**Issue by 0xtaosu** [slockit/DAO#284]
### How a cool idea!Why not continue it?

**Comment by MartinDelille** [comment]
Maybe because people lost so much money with this project that they don't want to hear about it?

**Comment by 0xtaosu** [comment]
@MartinDelille In my opinion , now the world is like this :The poor are poor and the rich are getting richer.With the development of the Internet ,more and more people will discover the truth about the uneven distribution of wealth. If we dont change it ,the world will be torn , Maybe we should continue it

**Comment by MartinDelille** [comment]
@taosue I personally lost 100$ in this project but I liked the idea and would be happy to see it work again. I’ll be happy to help as far as I can!

**Comment by ydennisy** [comment]
me too, has the vulnerability in the code been fixed?

**Comment by MartinDelille** [comment]
Yes: https://github.com/slockit/DAO/commit/3aafa36be722922014370f74ade55865335beb9b

**Comment by 0xtaosu** [comment]
@MartinDelille @ydennisy In my opinion，to design a DAO , first we need to do the Mechanism Design and then we think about the right data structure to implement and maintain our mechanism

**Comment by MartinDelille** [comment]
@taosue Is there already a mechanism design documentation about the current project?

**Comment by 0xtaosu** [comment]
@MartinDelille No , In fact even the economic theory of blockchain is still not mature , not to mention DAO

**Comment by MartinDelille** [comment]
@taosue Ok so how do we start? Is there some slockit people around? Should we fork the project?

**Comment by MartinDelille** [comment]
It seems no one got interested in the project since two years when looking at the stats: https://github.com/slockit/DAO/network

**Comment by 0xtaosu** [comment]
@MartinDelille Maybe we should learn from DAO's ideas and add our ideas , so we should have an more efficient discussion about mechanism design and blockchain

**Comment by MartinDelille** [comment]
@xalava What do you think about it?
